README.md

```diff
@@ -1 +1,40 @@
-# octopus-oc-template-gradle-plugin
\ No newline at end of file
+# octopus-oc-template-gradle-plugin
+This Gradle plugin provides a convenient way to interact with OKD/OpenShift templates in your build process. It wraps common oc CLI commands into easily callable Gradle tasks.
+
+### Features
+- Initialize and process OpenShift templates with parameters
+- Create resources from templates
+- Wait for pods to be ready
+- Fetch logs from pods
+- Clean up by deleting the created resources
+
+### Usage
+#### Apply the Plugin
+
+```kotlin
+plugins {
+   id("org.octopusden.octopus.oc-template")
+}
+```
+#### Register a Template Service
+```kotlin
+val testService = ocTemplateService.register("testService") {
+   namespace = "your-okd-namespace"             // Target OKD project namespace
```

Comment:
It would be nice to have the ability to control this parameter through an environment variable as well (i.e., if it’s not configured in the code, then check the environment variable).